docs/user-guide/creating-application/docker-build-configuration.md

```diff
@@ -1,59 +1,105 @@
+* [Store Container Image](https://docs.devtron.ai/usage/applications/creating-application/docker-build-configuration#store-container-image)
+* [Build the Container Image](https://docs.devtron.ai/usage/applications/creating-application/docker-build-configuration#build-the-container-image)
+* [Advanced Options](https://docs.devtron.ai/usage/applications/creating-application/docker-build-configuration#advanced-options)
 
-* **Image store**
-* **Checkout path**
-* **Advanced**
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/docker-build-configuration/build-configuration-latest1.jpg)
 
-## Image Store
-In Image store section, You need to provide two inputs as given below: 
-1. Docker registry
-2. Docker repository
+## Store Container Image
 
-### 1. Docker Registry
-Select the docker registry that you wish to use. This registry will be used to [store docker images](../global-configurations/docker-registries.md).
+The following fields are provided on the **Store Container Image** section:
 
-### 2. Docker Repository
-In this field, add the name of your docker repository. The repository that you specify here will store a collection of related docker images. Whenever an image is added here, it will be stored with a new tag version.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/docker-build-configuration/store-container-registry.jpg)
+
+| Field | Description |
+| --- | --- |
+| **Container Registry** | Select the container registry from the drop-down list or you can click **Add Container Registry**. This registry will be used to [store docker images](../global-configurations/docker-registries.md). |
+| **Container Repository** | Enter the name of your container repository, preferably in the format `username/repo-name`. The repository that you specify here will store a collection of related docker images. Whenever an image is added here, it will be stored with a new tag version. |
 
 **If you are using docker hub account, you need to enter the repository name along with your username. For example - If my username is *kartik579* and repo name is *devtron-trial*, then enter kartik579/devtron-trial instead of only devtron-trial.**
 
 ![](../../.gitbook/assets/docker-configuration-docker-hub.png)
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/docker-build-configuration/docker-build-config-container-registry1.png)
 
+## Build the Container Image
+
+In order to deploy the application, we must build the container images to configure a fully operational container environment.
+
+You can choose one of the following options to build your container image:
+* **I have a Dockerfile**
+* **Create Dockerfile**
+* **Build without Dockerfile**
+
+### Build Docker Image when you have a Dockerfile
+
+A `Dockerfile` is a file that you create which in turn produces a Docker image when you build it.
```

Comment:
A Dockerfile is a text document that contains all the commands a user could call on the command line to assemble an image.

Reply:
Incorporated. Resolved.